src/utils/debounce.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Returns a function that will not be triggered after right after a call, waiting ${wait} seconds
+ * to be able to invoke it again. If ${immediate}, it will call again in the leading edge, instead of trailing.
+ * It's a adapted version of {@link http://underscorejs.org/#debounce}
```

Comment:
And the library is going from 8.12 kB(custom debounce) to 17.6 kB (lodash debounce)
I've already made the changes to use lodash, so if is no problem I can update the PR.